src/test/java/core/basesyntax/service/CsvReaderServiceImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvReaderServiceImplTest {
+    private static final String FILE_FROM = "src/test/java/resources/fileWithCorrectData.csv";
+    private static final String WRONG_FILE_FROM = "src/test/java/resources/testTestTest.csv";
+    private static ReaderService readerService;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        readerService = new CsvReaderServiceImpl();
+    }
+
+    @Test
+    public void readFromFileTestWithCorrectInput_Ok() {
+        List<String> expected = new ArrayList<>();
+        try (BufferedReader readFile = new BufferedReader(new FileReader(FILE_FROM))) {
```

Comment:
You can user Files.readAllLines to shorten the code, it uses BufferedReader inside

Reply:
done